server/cmwell-fts/src/main/scala/cmwell/fts/FTSService.scala

```diff
@@ -989,30 +989,29 @@ class FTSService(config: Config) extends NsSplitter{
       val name = filter.name + "_" + counter
       counter += 1
       filtersMap.put(name, filter)
-
       val aggBuilder:AggregationBuilder = filter match {
         case TermAggregationFilter(_, field, size, _) =>
-          AggregationBuilders.terms(name).field(field).size(size)
+          AggregationBuilders.terms(name).field(field + ".%exact").size(size)
```

Comment:
the exact should be used only on text fields

Reply:
after re-thinking THIS IS NOT A CORRECT FIX for the problem since we change the behavior for stats by :.
as discussed with Yaakov and Dudi we will return 400 in a case of stats by text field.